src/stores/appInterface.ts

```diff
@@ -19,18 +19,18 @@ export enum SubMenuName {
  * Available sub menus names
  */
 export enum SubMenuComponentName {
-  SettingsGeneral = 'settings-general',
-  SettingsInterface = 'settings-interface',
-  SettingsJoystick = 'settings-joystick',
-  SettingsVideo = 'settings-video',
-  SettingsTelemetry = 'settings-telemetry',
-  SettingsAlerts = 'settings-alerts',
-  SettingsDev = 'settings-dev',
-  SettingsMission = 'settings-mission',
-  SettingsActions = 'settings-actions',
-  SettingsDataLake = 'settings-datalake',
-  ToolsMAVLink = 'tools-mavlink',
-  ToolsDataLake = 'tools-datalake',
+  SettingsGeneral = 'SettingsGeneral',
```

Comment:
I'm not qualified to review this code of course but I'm surprised that we're changing the formatting of all these lines.  I'd expect one new entry but I wouldn't have expected existing entries to change

Reply:
Style changes should at least break in another commit